PowerEditor/installer/themes/DarkModeDefault.xml

```diff
@@ -1444,5 +1444,9 @@ License:             GPL2
         <WidgetStyle name="Inactive tabs" styleID="0" fgColor="808080" bgColor="C0C0C0" />
         <WidgetStyle name="URL hovered" styleID="0" fgColor="A3DCA3" />
         <WidgetStyle name="Document map" styleID="0" fgColor="000000" bgColor="FFFFFF" />
+        <WidgetStyle name="Autocomplete list" styleID="0" fgColor="C0C0C0" bgColor="202020" fontStyle="0" />
+        <WidgetStyle name="Autocomplete selected item" styleID="0" fgColor="E0E0E0" bgColor="404040" fontStyle="0" />
+        <WidgetStyle name="Calltip" styleID="0" fgColor="808080" bgColor="202020" fontStyle="0" />
+        <WidgetStyle name="Calltip highlighted text" styleID="0" fgColor="E0E0E0" fontStyle="0" />
```

Comment:
The problem of using user customized settings is:
1. `DarkModeDefault.xml` is not updated by the installer, so these changes won't be taken into account by the new code when users update their Notepad++.
2. These settings work (go) only with `DarkModeDefault.xml`, if user switches to other themes, it will stop working.